manifests/kustomize/base/rbac.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tinkerbell-boots
+  namespace: tinkerbell
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Can this just be a `Role`?

Reply:
It probably should be. The issue with boots (and hegel, and tink) is that incoming requests don't encode a namespace, so it only really makes sense to operate the components in a single namespace